docs/admin_guide.md

```diff
@@ -38,13 +38,13 @@ Agents is enabled automatically when using the pre-installed version. If you've
 
 ### Basic configuration
 
-If you have an Enterprise license, upload it to unlock additional features. 
+If you have an Entry, Enterprise, or Enterprise Advanced license, upload it to unlock additional features.
```

Comment:
Nit: We could just remove "Entry" or we could use this reword, since there is no "entry license" to upload
```suggestion
If you have an Enterprise, or Enterprise Advanced license, upload it to unlock additional features. If you don't have a license but are running Mattermost Enterprise Edition, an Entry license will be automatically applied for you.
```